src/Header/Nav/links.json

```diff
@@ -2,7 +2,25 @@
 	{
 		"href": "https://pathways.nice.org.uk/",
 		"text": "NICE Pathways",
-		"id": "pathways"
+		"id": "pathways",
+		"links": [
+			{
+				"text": "Browse",
+				"href": "/#browse"
```

Comment:
do we need forward slash here due to the hash?

Reply:
Yes, we want to link the browse section on the Pathways homepage